### PR TITLE
V1

### PR DESCRIPTION
**Pull request by selfish**
## Background and Notable Changes
- v1 of reviewdog action

### CODE REVIEW

File: ember-template-lint-formatter-rdjson/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  data += chunk;
+});
+
+process.stdin.on('end', function() {
+  const parsed = JSON.parse(data);
+  const rdJson = buildRdJsonOutput(parsed);
+  console.log(JSON.stringify(rdJson));
+});
+
+// const data = require('./test-fix.json');
```
Comment (taltcher) [inline ember-template-lint-formatter-rdjson/index.js:85]: Can be removed :-)

File: ember-template-lint-formatter-rdjson/out.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
```
Comment (taltcher) [inline ember-template-lint-formatter-rdjson/out.json]: Need to be removed

File: ember-template-lint-formatter-rdjson/test-fix.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (taltcher) [inline ember-template-lint-formatter-rdjson/test-fix.json]: Need to be removed

File: ember-template-lint-formatter-rdjson/index.js (hunk 1 above)
Comment (selfish) [reply ember-template-lint-formatter-rdjson/index.js:85]: Leaving for debug

File: ember-template-lint-formatter-rdjson/out.json (hunk 2 above)
Comment (selfish) [reply ember-template-lint-formatter-rdjson/out.json]: Leaving for debug

**Review by taltcher — APPROVED** [review]

**Pull request merged**